File: conformance-tests/base-types/template-literals/multiline-escape-sequences-with-expressions_m5.lua
local value = ([[

backspace       --> abc%s]] .. "\b%s" .. [[def
form feed       --> abc%s]] .. "\f%s" .. [[def
new line        --> abc%s]] .. "\n%s" .. [[def
carriage return   --> abc%s]] .. "\r%s" .. [[def
horizontal tab  --> abc%s]] .. "\t%s" .. [[def
vertical tab    --> abc%s]] .. "\v%s" .. [[def
zero            --> abc%s]] .. "\0%s" .. [[def
single quote    --> abc%s'%sdef
double quote    --> abc%s"%sdef
backtick        --> abc%s`%sdef
backspace       --> abc%s]] .. "\\%s" .. [[def

-- regular characters but escaped

a
c
d
e
g
]]):format(
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar),
	tostring(foo),
	tostring(bar)
)

File: conformance-tests/base-types/template-literals/multiline-escape-sequences-with-expressions_m5.js
const value = `
backspace       --> abc${foo}\b${bar}def
form feed       --> abc${foo}\f${bar}def
new line        --> abc${foo}\n${bar}def
carriage return   --> abc${foo}\r${bar}def
horizontal tab  --> abc${foo}\t${bar}def
vertical tab    --> abc${foo}\v${bar}def
zero            --> abc${foo}\0${bar}def
single quote    --> abc${foo}\'${bar}def
double quote    --> abc${foo}\"${bar}def
backtick        --> abc${foo}\`${bar}def
backspace       --> abc${foo}\\${bar}def

-- regular characters but escaped

\a
\c
\d
\e
\g
`;
